satpy/readers/hrit_jma.py

```diff
@@ -283,6 +297,42 @@ def _mask_space(self, data):
         geomask = get_geostationary_mask(area=self.area)
         return data.where(geomask)
 
+    def _get_acq_time(self):
+        """
+        Acquisition times for a subset of scanlines are stored in the header
+        as follows:
+
+        b'LINE:=1\rTIME:=54365.022558\rLINE:=21\rTIME:=54365.022664\r...'
+
+        Missing timestamps in between are computed using linear interpolation.
```

Comment:
I think this information should also be available at module-level documentation, so that the user doesn't have to search the code for how this is done.

Reply:
Good idea, will do!